test/test_logging.py

```diff
@@ -0,0 +1,146 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Test Nature logging methods
+"""
+from typing import List, Dict, Set
+import sys
+import unittest
+import logging
+import tempfile
+import contextlib
+import io
+import os
+from test import QiskitNatureTestCase
+from qiskit_nature import logging as nature_logging
+
+
+class TestHandler(logging.StreamHandler):
+    """Unit Test Handler"""
+
+    def __init__(self):
+        """
+        Initialize the handler.
+        """
+        super().__init__(sys.stdout)
+        self.records: List[logging.LogRecord] = []
+
+    def emit(self, record) -> None:
+        """handle record"""
+        self.records.append(record)
+
+
+class TestLogging(QiskitNatureTestCase):
+    """Test logging"""
+
+    def setUp(self):
+        super().setUp()
+        self._test_handler = TestHandler()
+        nature_logging.dict_aux_operators = True
```

Comment:
If you want to avoid the `DeprecationWarnings` currently raised if this is False you now need to update this to actually use the settings:
```suggestion
        nature_settings.dict_aux_operators = True
```
Andd add `from qiskit_nature import settings as nature_settings` at the top of the file :+1: